src/driver/Types.fs

```diff
@@ -65,6 +65,8 @@ type EngineParameters =
         /// Specifies whether to run results analyzer.
         runResultsAnalyzer : bool
 
+        /// File path to the client certificate.
+        certFilePath : string
```

Comment:
>string [](http://example.com/codeflow?start=23&length=6)

Could you please make this a string option, and set the default to None below?  Then, in Program.fs you can check for the None value so the parameter isn't sent to the engine at all when the user does not provide a cert.